app/views/drawings/_form.html.haml

```diff
@@ -29,6 +29,8 @@
 
               = f.input :mood_rating, type: 'number', required: true, label: false, placeholder: 'Mood rating (1 to 10)'
             .form-group.text-center
+              = f.input :country, priority: ["GR", "HU", "RS"], selected: (current_user.country if @drawing.id == nil), label: false, include_blank: "Country picture was drawn in", class: 'form-control'
```

Comment:
We can check if this is a new record with the method `@drawing.new_record?` instead of using the nil check